frontend/src/app/main/ui/ds/controls/combobox.cljs

```diff
@@ -69,7 +70,7 @@
 (mf/defc combobox*
   {::mf/props :obj
    ::mf/schema schema:combobox}
-  [{:keys [id options class placeholder disabled has-error default-selected on-change] :rest props}]
+  [{:keys [id options class placeholder disabled has-error default-selected on-change maxlength] :rest props}]
```

Comment:
should it be `max-length` instead of `maxlength` ?

Reply:
The HTML attribute for inputs is named `maxlength` and since it should be used as a default input I think it is better to keep it as the standard, don't you think?